vfcompositor/Compositor.c

```
@@ -42,6 +44,17 @@ static TaskButton g_task_buttons[MAX_TASK_BUTTONS];
 static int g_task_button_count = 0;
 static Window* g_start_menu_window = NULL;
 
+// Deferred destruction to avoid race with render loop
+static Window* g_pending_destroy[MAX_WINDOWS];
```

Clang (14.0.6)

[warning] variable 'g_pending_destroy' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)